icinga2.spec

```diff
@@ -347,7 +347,10 @@ export CC=gcc-4.8
 export CXX=g++-4.8
 %endif
 
-%{?scl_enable} cmake $CMAKE_OPTS -DCMAKE_C_FLAGS:STRING="%{optflags} %{?march_flag}" -DCMAKE_CXX_FLAGS:STRING="%{optflags} %{?march_flag}" .
+%{?scl_enable} cmake $CMAKE_OPTS \
+  -DCMAKE_C_FLAGS:STRING="%{optflags} %{?march_flag}" \
+  -DCMAKE_CXX_FLAGS:STRING="%{optflags} %{?march_flag}" \
+  -DICINGA2_BUILD_HOST_NAME:STRING=hostname .
```

Comment:
This will render the output in `icinga2 --version` useless as everyone uses the same static string. I'm strongly against this change for our upstream packages.

Reply:
In theory it could be any string. Is there one you would prefer, like `icinga2` or `rpmbuild`?

It would also only affect rpm builds.

Is the information about the build node under `--version` more important than reproducible builds?

Reply:
It helps to determine which host actually has built the rpm binaries, whether official sources or some custom package repository. I don't see why reproducible builds should overrule this specific information, this seems to be a SUSE-specific requirement.

Reply:
You can only have reproducible builds if everything is static. Otherwise the checksum will be different when the name of the host changes. Maybe we should just make sure that all build hosts have the same hostname. I will check this out.

Reply:
If it is urgent (because of the 2.9.0 release) I can create a new pr without this patch or revert it for now.